src/main/kotlin/com/terraformation/backend/tracking/model/AssignedPlotDetails.kt

```diff
@@ -15,4 +16,33 @@ data class AssignedPlotDetails(
     val plantingSubzoneId: PlantingSubzoneId,
     val plantingSubzoneName: String,
     val plotName: String,
-)
+) {
+  fun gpxWaypoints(): List<GpxWaypoint> {
+    return listOf(
+        GpxWaypoint(
+            boundary.coordinates[SOUTHWEST].y,
+            boundary.coordinates[SOUTHWEST].x,
+            "$plotName Southwest Corner"),
```

Comment:
Is it acceptable to have waypoint names in English if the user is using the app in Spanish? (Maybe a product question; I'm not sure if the names are even user-visible at all.)

Reply:
They will be user-visible in a GPS device. I think the shorthanded direction (SW, NW etc....) is likely a lot easier but let me double check with product.

Reply:
In that case they'll need to be localizable, I'm betting. The French word for "west" is "ouest," so the French abbreviation for "northwest" is "NO" rather than "NW."